ehrapy/preprocessing/_bias.py

```diff
@@ -0,0 +1,119 @@
+from collections.abc import Iterable
+from typing import Literal
+
+import numpy as np
+import pandas as pd
+from anndata import AnnData
+
+from ehrapy import logging as logg
+from ehrapy.anndata import anndata_to_df
+
+
+def bias_detection(
+    adata: AnnData,
+    sensitive_features: Iterable[str] | Literal["all"],
+    corr_threshold: float = 0.5,
+    smd_threshold: float = 0.5,
+    feature_importance_threshold: float = 0.1,
+    prediction_confidence_threshold: float = 0.5,
+    corr_method: Literal["pearson", "spearman"] = "spearman",
+):
+    """Detects bias in the data.
+
+    Args:
+        adata: An annotated data matrix containing patient data.
```

Comment:
```suggestion
        adata: An annotated data matrix containing EHR data.
```

I know that we have plenty of variants of that in the code and they're all equally bad, but I want to avoid `patient` because it implies a certain structure of the observations (patient vs patient visit view and other things)